proto/one/enterprise/data/report_definition_json_v2.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+import "enterprise_data_table_query_parameter.proto";
+import "post_filtering.proto";
+
+option csharp_namespace ="ONE.Models.CSharp.Enterprise.Data";
+
+enum ReportDefinitionCategory {
+    REPORT_DEFINITION_CATEGORY_UNDEFINED = 0;
+    REPORT_DEFINITION_CATEGORY_SPREADSHEET = 1;
+    REPORT_DEFINITION_CATEGORY_SAMPLE_MANAGER = 2;
+}
+
+message ReportDefinitionJsonV2 {
```

Comment:
Just occurred to me but do you need to add this to the Content protobuf so we can send it in an apiResponse?

Reply:
No, these new messages won't be returned by an api request. It defines the structure of the ReportDefintionJson property of the ReportDefinition for version 2 report definitions.